cmake/README.md

```diff
@@ -1,28 +1,36 @@
 ## CMake for Mbed OS
 
-Until further notice, CMake for Mbed OS is still in development. 
+In order to [replace the traditional Mbed tools](https://os.mbed.com/blog/entry/Introducing-the-new-Mbed-Tools/), we are adding CMake support to allow the generation of build systems instructions to build Mbed OS as a library.
+Mbed OS is built as collection of all libraries supported by a given Mbed target. Application project executables can be linked with the monolithic `mbed-os` library to build Mbed applications.
+
+We are currently working to break down the `mbed-os` library into several individual libraries that applications can selectively choose to include depending on their needs.
+
+The following features are not yet supported and will be progressively added:
+* Application profile selection (`bare metal`)
+* C library selection (`small`)
+* `s/f/v/printf` library selection (`minimal-printf`)
+* Other features that require altering compiler/linker command line options
```

Comment:
> Until further notice, CMake for Mbed OS is still in development. 

Do others read this update the same way as the previous message? I wanted to emphasize this CMake work is  "experimental" .
I do not want to set expectations that someone takes this work and start complaining this is not functional or similar.

Something similar:

`CMake is in experimental stage and it is still in development. It does not yet completely replace our current tools. If you find a bug, please report or better, send a new pull request. Contributions are welcome!`

Reply:
Please review [this](https://github.com/ARMmbed/mbed-os/compare/bb73268fddb99f555b85900fe4a23a053204fa13..a476cb2509e8451ec508071086401e68b22aee14).

Reply:
It looks fine to me (one typo: raise instead of raised)

Reply:
[Updated](https://github.com/ARMmbed/mbed-os/compare/e01d0b16525ceb906bdd445d9f4c247433505d6d..a7f130bc06ff87104dad49647a203291c5bb6917).